<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="3" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="5651" started-at="2019-12-31T05:47:37Z" finished-at="2019-12-31T05:47:43Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="5651" started-at="2019-12-31T05:47:37Z" finished-at="2019-12-31T05:47:43Z">
      <class name="baseobject.dress2">
        <test-method status="PASS" signature="dresses()[pri:0, instance:baseobject.dress2@5ab956d7]" name="dresses" duration-ms="1037" started-at="2019-12-31T05:47:37Z" finished-at="2019-12-31T05:47:38Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- dresses -->
        <test-method status="PASS" signature="verifdisplay()[pri:0, instance:baseobject.dress2@5ab956d7]" name="verifdisplay" duration-ms="1502" started-at="2019-12-31T05:47:38Z" finished-at="2019-12-31T05:47:40Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifdisplay -->
        <test-method status="FAIL" signature="verifycart()[pri:0, instance:baseobject.dress2@5ab956d7]" name="verifycart" duration-ms="3060" started-at="2019-12-31T05:47:40Z" finished-at="2019-12-31T05:47:43Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at baseobject.dress2.verifycart(dress2.java:36)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifycart -->
      </class> <!-- baseobject.dress2 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
